.github/workflows/ci.yml

```diff
@@ -74,6 +74,26 @@ jobs:
         run: |
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
           $MAVEN clean install -B --strict-checksums -V -T 1C -DskipTests -P ci -pl '!:trino-server-rpm'
+      - name: Remove Trino from local Maven repo to avoid caching it
+        # Avoid caching artifacts built in this job, cache should only include dependencies
+        if: steps.cache.outputs.cache-hit != 'true'
+        run: rm -rf ~/.m2/repository/io/trino/trino-*
+
+  server-checks:
```

Comment:
Did you omit `timeout-minutes` intentionally? By the way, this job looks pretty slow. It 60m in https://github.com/trinodb/trino/actions/runs/4606301570/jobs/8139432887?pr=16871

Reply:
I want to extend it, since 45 minutes wasn't enough, but no definite value comes to mind.
The time required depends on Ubuntu repositories, let's start with 60 minutes and iterate from there.

Reply:
> > let's start with 90 minutes and iterate from there.
> 
> 90 is definitely way more than we want all the CI jobs to complete in. are we talking about some overly heavy product tests?

This is simply time to prepare the RPM and Docker images for multiple architectures, and see if Trino starts up.
I've seen in take over 60 minutes when one of the Ubuntu mirrors is restarted, but in optimal conditions it should take about 30 minutes. It's a question of how resilient we want our CI to be and how much time are we willing to spend trying in sub-optimal conditions.